timelock-agent/src/main/java/com/palantir/timelock/paxos/AsyncTimeLockServicesCreator.java

```diff
@@ -78,7 +77,6 @@ public TimeLockServices createTimeLockServices(
 
         AsyncTimelockResource asyncTimelockResource =
                 new AsyncTimelockResource(maybeEnhancedLockLog, asyncTimelockService);
-        LockWatchingResource lockWatchingResource = new LockWatchingResource(asyncTimelockService);
```

Comment:
~not too sure about this, I think you still want stuff behind the `ATR`, but I'll think about it whilst goign through the rest of the PR.~

Turns out `ATR` extends `LWS` so it's all good.